tools/register-arns-ants.js

```diff
@@ -0,0 +1,32 @@
+import { IO, AOProcess, ArweaveSigner, createAoSigner } from '@ar.io/sdk';
+import Arweave from 'arweave';
+
+const registryId = 'CKcXPRsQCsmNf6Wnx0njnlf2upecRyk35d8LWGk85AQ';
+
+async function main() {
+  const arweave = Arweave.init();
+  const jwk = await arweave.wallets.generate();
+  const signer = createAoSigner(new ArweaveSigner(jwk));
+  const ario = IO.init();
+  const arnsRecords = await ario.getArNSRecords({
+    limit: 50_000,
+  });
+  const antIds = arnsRecords.items.map((record) => record.processId);
+  const processIds = new Set(antIds);
+
+  const registry = new AOProcess({ processId: registryId });
+  let count = 0;
+
+  for (const antId of [...processIds]) {
+    console.log(`Registering ${count++}/${processIds.size}...`);
+    await registry.send({
+      tags: [
+        { name: 'Action', value: 'Register' },
+        { name: 'Process-Id', value: antId },
+      ],
+      signer,
+    });
```

Comment:
nit - add a catch to avoid blowing up on a one-off failure (we already did it so not a big deal)